src/main/java/cc/hyperium/config/Settings.java

```diff
@@ -403,10 +390,12 @@
     public static String PAUSE_STYLE = GuiStyle.HYPERIUM.name();
 
     @ConfigOpt
+    @ToggleSetting(name = "gui.spotify.spotifynotifications", category = SPOTIFY)
     public static boolean SPOTIFY_NOTIFICATIONS = false;
 
     @ConfigOpt
-    public static boolean SPOTIFY_FORCE_DISABLE = true;
+    @ToggleSetting(name = "gui.spotify.forcedisable", category = SPOTIFY)
+    public static boolean SPOTIFY_FORCE_DISABLE = false;
```

Comment:
I don't think the force disable option should exist anymore since spotify works now.

Reply:
existed before it broke, why not keep it